exp-player/addon/components/exp-lookit-dialogue-page/component.js

```diff
@@ -0,0 +1,688 @@
+import Ember from 'ember';
+import layout from './template';
+import ExpFrameBaseUnsafeComponent from '../../components/exp-frame-base-unsafe/component';
+import FullScreen from '../../mixins/full-screen';
+import VideoRecord from '../../mixins/video-record';
+
+let {
+    $
+} = Ember;
+
+/**
+ * @module exp-player
+ * @submodule frames
+ */
+
+/**
+ * Frame to implement a storybook page with dialogue spoken by characters.
+ * First, characters appear and any main narration audio is played.
+ * Next, the user can click on the characters to play additional audio
+ * associated with each character image, or (for a choice trial) the user clicks
+ * one of the images to select it as an answer. Once main narration audio has
+ * been played and either a selection has been made (for a choice trial,
+ * isChoiceFrame: true) or all
+ * required character audio has been played (for a non-choice trial), the user
+ * can proceed by pressing 'next'. (A trial with only main narration audio can
+ * also simply auto-proceed when audio is finished.)
+ *
+ * Recording is optional. If webcam recording is conducted (doRecording: true)
+ * then audio does not start until recording does, to ensure the entire trial
+ * is recorded.
+ *
+ * The character images are specified in 'images', including an image source,
+ * positioning on the screen, any animation at the start of the trial, any
+ * associated audio, and whether that audio is required.
+ *
+ * This frame extends ExpFrameBaseUnsafe because it is displayed fullscreen
+ * and expected to be repeated.
+
+```json
+ "frames": {
+         "phase-1": {
+            "backgroundImage": "https://s3.amazonaws.com/lookitcontents/politeness/img/order1_test1_background.png",
+            "doRecording": true,
+            "autoProceed": false,
+            "parentTextBlock": {
+                "title": "Parents!",
+                "text": "some instructions",
+                "emph": true
+            },
+            "images": [
+                {
+                    "id": "protagonist",
+                    "src": "https://s3.amazonaws.com/lookitcontents/politeness/img/order1_test1_listener1.png",
+                    "left": "40",
+                    "bottom": "2",
+                    "height": "60"
+                }
+            ],
+            "kind": "exp-lookit-dialogue-page",
```

Comment:
Good point! Here I do want to show two distinct usage examples, but I've made them shorter and now describe what each one does.